Documentation/Blazorise.Docs/Pages/Docs/Components/Modals/ModalPage.razor

```diff
@@ -103,6 +103,51 @@
     Modal dialogs are disruptive by nature and should be used sparingly. Do not use them to communicate nonessential information, such as success messages like "Logged in", "Copied", etc. Instead, use <Anchor To="docs/services/notification-provider">Notifications</Anchor> when appropriate.
 </DocsPageParagraph>
 
+<Heading Size="HeadingSize.Is3">
+    Fullscreen Mode
+</Heading>
+
+<DocsPageParagraph>
+    When using the Fullscreen mode for the Modal component, it's crucial to be aware of how you structure your layout within the <Strong>ModalBody</Strong>.
+</DocsPageParagraph>
+
+<DocsPageParagraph>
```

Comment:
Generally the base documentation pages show case examples of the features available & usage. Sometimes with well delimited notes about certain quirks (usage of Alerts).

It seems like we have much to explain here, so using just an Alert would not be enough. Would this be best on  "Best practices" section or something?

Reply:
Yes, this is part of **Best Practices**. While it is somewhat long I think it gives enough information about the context of a problem.

---

In the long run, we should expand all our existing examples with more descriptive use cases and best practices. Not related to this PR, but the usage of TextEdit and other inputs without wrapping them with Field comes to mind.